lib/control/v1/signature.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-nested-callbacks */
+
+'use strict';
+
+const CP = require('child_process');
+const Path = require('path');
+const HTTP = require('http');
+
+const FORBIDDEN = 403;
+const defaultPort = 9806;
+const defaultMaxRetires = 5;
+
+class Server {
+  constructor(port, path, options) {
+    options = options || {};
+
+    this.port = Number(port) || defaultPort;
+    this.path = path || Path.resolve(__dirname, '../../../pkcs7/bin/server');
+    this.retries = 0;
+    this.maxRetires = Number(options.maxRetires) || defaultMaxRetires;
+  }
+
+  start() {
+    this.process = CP.spawn(this.path, [`-p ${this.port}`]);
```

Comment:
I left the path alone because It will be easier to figure out where the path is defined.
Port was changed to make more sense per your comment.